src/Microsoft.Diagnostics.Monitoring.HostingStartup/ParameterCapturing/ParameterCapturingService.cs

```diff
@@ -32,6 +32,13 @@ internal sealed class ParameterCapturingService : BackgroundService, IParameterC
 
         private readonly ILogger? _logger;
 
+        private static readonly string[] s_typeDenyList = {
```

Comment:
Maybe add some comments here to denote this is a parent namespace or parent type and partials are ok along the ./+ boundaries only.